src/utils/helpers.ts

```diff
@@ -172,3 +173,58 @@ export function makeId(length: number = 8): string {
   }
   return result;
 }
+
+export const roundToSmaller = (amount: any, decimals: number): string => {
+  const bn = bignumber(amount);
+  let [integer, decimal] = bn.toFixed(128).split('.');
+
+  if (decimal && decimal.length) {
+    decimal = decimal.substr(0, decimals);
+  } else {
+    decimal = '0'.repeat(decimals);
+  }
+
+  if (decimal.length < decimals) {
+    decimal = decimal + '0'.repeat(decimals - decimal.length);
+  }
+
+  if (decimal !== '') {
+    return `${integer}.${decimal}`;
+  }
+  return `${integer}`;
+};
+
+export const fromWei = (amount: any, unit: Unit = 'ether') => {
+  let decimals = 0;
+  switch (unit) {
+    case 'ether':
+      decimals = 18;
+      break;
+    default:
+      throw new Error('Unsupported unit (custom fromWei helper)');
+  }
+  return roundToSmaller(bignumber(amount || '0').div(10 ** decimals), decimals);
+};
+
+export const toWei = (amount: any, unit: Unit = 'ether') => {
+  let decimals = 0;
+  switch (unit) {
+    case 'ether':
+      decimals = 18;
+      break;
+    default:
+      throw new Error('Unsupported unit (custom toWei helper)');
+  }
+
+  return roundToSmaller(bignumber(amount || '0').mul(10 ** decimals), 0);
+};
+
+export const numberFromWei = (amount: any, unit: Unit = 'ether') => {
+  return Number(fromWei(amount, unit));
+};
```

Comment:
I think these functions should be already in the codebase with same name, expect for numberFromWei - it may have another name